sass/components/_emoticons.scss

```diff
@@ -268,13 +268,35 @@
     border-color: transparent !important;
     margin: 12px 12px 0 12px;
 
+    .skin-tones-animation {
+        &-enter {
+            transform: translateX(100%);
+        }
+
+        &-enter-active {
+            transform: translateX(0%);
+            transition: transform 200ms ease-in-out;
+        }
+
+        &-exit {
+            position: absolute;
+            width: 100% !important;
```

Comment:
if you add the width prop in the other steps you won't need to use the !important tag. We are trying to stay away from using it

Reply:
got it, I will take a look